envs/quadrotors/quad_multi_model.py

```diff
@@ -0,0 +1,60 @@
+import torch
+from torch import nn
+
+from algorithms.appo.model_utils import get_obs_shape, nonlinearity, create_standard_encoder, EncoderBase, \
+    register_custom_encoder
+from algorithms.utils.pytorch_utils import calc_num_elements
+from utils.utils import log
+
+
+class QuadMultiMeanEncoder(EncoderBase):
+    '''Mean embedding encoder based on the DeepRL for Swarms Paper'''
+    def __init__(self, cfg, obs_space, timing, self_obs_dim=18, neighbor_obs_dim=6, neighbor_hidden_size=32):
```

Comment:
Again, we don't need `obs_space` here, since we the main idea to pass obs_space here is to get observation shape, (take a look at doom_model.py). But here, since we already have self_obs_dim and neighbor_obs_dim, we don't need obs_space anymore.